SpectralCubeReprojectMaskExample_v3.ipynb

```diff
@@ -0,0 +1,1546 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Calculate noise map based on flat part of spectrum\n",
+    "cube_sclip_cut = cube_sclip.spectral_slab(60*u.km/u.s, 100*u.km/u.s)\n",
+    "mad_std_spectrum_sclip_cut = cube_sclip_cut.mad_std(axis=(1, 2))\n",
+    "plt.plot(mad_std_spectrum_sclip_cut.spectral_axis.value, mad_std_spectrum_sclip_cut.value, \n",
+    "         drawstyle='steps-mid')\n",
+    "plt.xlabel('Velocity (km/s)')\n",
+    "plt.ylabel(r' Noise standard deviation $\\sigma$ (K)')\n",
+    "mad_std_map_sclip = cube_sclip_cut.mad_std(axis=0) # Calculate sigma along the spectral dimension"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we'll create a \"plain mask\" where we mask the template cube and only keep pixels that are at or above 3 sigma (where sigma is given on a pixel-by-pixel basis by the noise map we just created)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Calculate and apply plain mask\n",
+    "plain_mask = cube1vel >= 3 * mad_std_map_sclip # Get plain 3sigma mask\n",
+    "plain_masked_slab = cube1vel.with_mask(plain_mask) # Mask the template molecule cube"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then, we'll apply that mask to the template molecule cube and create a more sophisticated \"signal mask\" from that cube, which takes into account the fact that connected regions of signal are more likely to be real than an individual high-sigma pixel."
```

Comment:
Link to the other signal masking tutorial here

Reply:
https://mybinder.org/v2/gh/radio-astro-tools/tutorials/HEAD?filepath=masking_and_moments%2Fmasking_and_moments_guide.ipynb